Makefile

```diff
@@ -69,12 +70,12 @@ $(CCTZ_LIB): $(CCTZ_OBJS)
 install: install_hdrs install_lib
 
 install_hdrs: $(CCTZ_HDRS)
-	$(SUDO) mkdir -p $(PREFIX)/include
-	$(SUDO) cp -p $? $(PREFIX)/include
+	$(SUDO) mkdir -p $(DESTDIR)/$(PREFIX)/include
```

Comment:
What is it that DESTDIR is buying us over PREFIX?  They always appear together.  Should we just rename PREFIX to DESTDIR?